editoast/src/models/train_schedule.rs

```diff
@@ -42,30 +57,40 @@ use super::{
 #[derivative(Default)]
 pub struct TrainSchedule {
     #[diesel(deserialize_as = i64)]
+    #[schema(value_type = i64)]
     pub id: Option<i64>,
     pub train_name: String,
+    #[schema(value_type = Vec<String>)]
     pub labels: DieselJson<Vec<String>>,
     pub departure_time: f64,
     pub initial_speed: f64,
-    #[derivative(Default(value = "DieselJson(Default::default())"))]
+    #[derivative(Default)]
+    #[schema(value_type = Vec<Allowance>)]
     pub allowances: DieselJson<Vec<Allowance>>,
     #[derivative(Default(
         value = "crate::schema::rolling_stock::RollingStockComfortType::default().to_string()"
     ))]
+    #[schema(value_type = RollingStockComfortType)]
     pub comfort: String,
```

Comment:
Can’t we have RollingStockComfortType as the type for comfort ?

Reply:
Sadly no, Diesel doesn't support enums. There are packages that seem to fill the gap but we didn't like any of those. So we planned to add enum support in ModelV2, but I've just found that crate that seems really nice: https://github.com/adwhit/diesel-derive-enum